opentelemetry-api/src/opentelemetry/context/context.py

```diff
@@ -30,24 +30,15 @@ class RuntimeContext(ABC):
 
     @abstractmethod
     def attach(self, context: Context) -> object:
-        """Sets the current `Context` object. Returns a
```

Comment:
Why remove the documentation? It is very possible that this documentation applies for all implementations of this abstract method.

Also this:

```python
from abc import ABC, abstractmethod
from inspect import getdoc


class Parent(ABC):
    @abstractmethod
    def method(self):
        """
        The documentation for method
        """


class Child(Parent):

    def method(self):
        pass


print(getdoc(Child().method))
```
```
The documentation for method
```

Reply:
Since `_RuntimeContext` is used internally, we do not expect anyone to inherit it and implement a context on their own. As well, since we are hiding the class via protected, the documentation will not show up in readthedocs anyways.

Reply:
Still nice to have for us so if you mouse a call site, IDEs will show the docstring

Reply:
Sure, I added to both.